commcare_connect/opportunity/migrations/0078_create_prod_hq_server.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.2.5 on 2025-06-23 14:49
+
+
+from django.conf import settings
+from django.db import migrations
+from oauth2_provider.models import Application
+
+
+def create_prod_hq_server_and_update_objects(apps, schema_editor):
+    HQServer = apps.get_model("opportunity", "HQServer")
+    Opportunity = apps.get_model("opportunity", "Opportunity")
+    HQApiKey = apps.get_model("opportunity", "HQApiKey")
+    CommCareApp = apps.get_model("opportunity", "CommCareApp")
+
+    oauth_application = Application.objects.first()
```

Comment:
I don't think ordering is technically guaranteed by postgres. If this is an existing application, it would be safer to fetch by `id`

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/666/commits/e8c25ff58ed227d01292032b64b7e0c375d77e1c